src/ipx.ts

```diff
@@ -37,6 +38,8 @@ export interface IPXOptions {
   domains?: false | string[];
   alias: Record<string, string>;
   fetchOptions: RequestInit;
+  cache?: boolean;
+  cacheMetadataStore?: Storage;
```

Comment:
storage variable name might needed to change in case to match "global" storage scope for input and output

dont really know overall package future structure and whats was planned